Master

anjisag opened the pull request.

dogriffiths committed:
Source code for chapter 14: Material Design.

We make one more set of amendments to the BitsAndPizzas app.
This time we convert the interface to use RecyclerViews and
CardViews.

dogriffiths committed:
Corrected issue that hid action bar icons.

There was previously a problem with the `BitsAndPizzas` app which
meant that icons could disappear from the action bar, leaving only
text menus. This change corrects the namespace used in the
`main_menu.xml` file.

dogriffiths commented:
Hello,

Could you provide a quick summary of the changes you’d like me to pull back?

Thanks.

DG

dogriffiths committed:
Fix bug that left the odometer service running.

The bound odometer service previously kept running when the associated
activity closed. The change no stops the service when the app is left.

dogriffiths committed:
Fix rotation bug in the workout app reported by j8s0n.

Previously, the back button in the workout app would not work
correctly if there were multiple items in the back stack. The code was
erroneously creating transactions and storing state in the
=onCreateView= method, rather than in =onCreate=.

dogriffiths committed:
Changed a layout text size from dp to sp.

dogriffiths committed:
Added an open source license

dogriffiths committed:
Upgrade to Android Studio 2. Issue reported by @tomjohnson1492

Previously, if you opened one of the projects in Android Studio 2, you
would get a gradle error:

    Error:Gradle version 2.10 is required. Current version is 2.2.1.
    If using the gradle wrapper, try editing the distributionUrl in
    .../gradle-wrapper.properties to gradle-2.10-all.zip
    Fix Gradle wrapper and re-import project
    Gradle settings

This change fixes the gradle wrapper to use version 2.10.

dogriffiths committed:
chore: Upgrade to gradle 2.3.2

dogriffiths committed:
refactor: Remove deprecated attribute

dogriffiths committed:
refactor: Remove unnecessary semicolon

dogriffiths committed:
refactor: Add override annotation

dogriffiths committed:
refactor: Add line break
